projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -5041,7 +5041,10 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
             return 0;
         }
 
-        this.endEdit(false);
+        const editModeCell = this.crudService.cell;
```

Comment:
I don't think this is necessary. Maybe remove the call to `refreshSearch()` in the `escape_editMode()` API service method.
There were changes to the highlight directive cdr cycle and I believe calling `refreshSearch` is not needed in most instances.